### PR TITLE
Validators: uses Ethereum address for proof of possession

### LINKED ISSUE #449: Validators SBAT submit their BLS signed ethereum address as proof-of-possession
### Expected Behavior

Validators prove possession of their private key by signing their ethereum address

### Current Behavior

Validators prove possession of their private key by signing their BLS public key

### PR DESCRIPTION
**Pull request by kobigurk**
### Description

Currently, the proof of possession is used with the BLS public key, opening the possibility for front-running attacks. This changes the behaviour such that the Ethereum address is used.

### Tested

Runs end-to-end tests with the new method.

### Other changes

Upgrades to the latest Blake2x-based hashing algorithm in bls-zexe.

### Related issues

- Fixes https://github.com/celo-org/celo-blockchain/issues/449

### Backwards compatibility

Not backwards compatible.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -4647,6 +4647,46 @@
     source-map-support "^0.5.7"
     tslib "^1.9.3"
 
+"@stablelib/binary@^0.7.2":
```
Comment (nategraf) [inline yarn.lock:4650]: Is there a requirement for these new packages for this PR?

**Review by nategraf — APPROVED** [review]
👍

File: yarn.lock (hunk 1 above)
Comment (kobigurk) [reply yarn.lock:4650]: I need the @stablelib/blake2xs package, which I guess pulls all of these.

**Review by kevjue — APPROVED** [review]
LGTM!

**Pull request merged**